lib/trento_web/controllers/v1/prometheus_controller.ex

```diff
@@ -1,18 +1,42 @@
 defmodule TrentoWeb.V1.PrometheusController do
   use TrentoWeb, :controller
 
+  use OpenApiSpex.ControllerSpecs
+
   alias Trento.Integration.Prometheus
+  alias TrentoWeb.OpenApi.Schema
 
   require Logger
 
   action_fallback TrentoWeb.FallbackController
 
+  operation :targets,
+    summary: "Get Prometheus exporters targets",
+    tags: ["Target Infrastructure"],
+    description: "Get Prometheus targets with the Http Discovery format",
+    responses: [
+      ok:
+        {"A collection of HttpSTD targets", "application/json",
+         TrentoWeb.OpenApi.Schema.HttpStd.TargetList}
+    ]
+
   def targets(conn, _) do
     targets = Prometheus.get_targets()
 
     render(conn, "targets.json", targets: targets)
   end
 
+  operation :exporters_status,
+    summary: "Get prometheus exporters status",
+    tags: ["Target Infrastructure"],
+    description: "Get Prometheus exporters status for a host identified by host id",
+    responses: [
+      ok:
+        {"The status for the prometheus exporter", "application/json",
+         TrentoWeb.OpenApi.Schema.Prometheus.ExporterStatus},
+      not_found: Schema.NotFound.response()
```

Comment:
We should maybe have other error besides not found?
For example, if the prometheus server doesn't exists or something like that (in this case we return 500 right now).
The thing is that we don't check if the coming host id even exists and maybe we should do it. Otherwise we depend on parsing the prometheus server response

Reply:
I have just checked, and if we use a host id that is not registered in our server, we will receive an empty map with 200 return code

Reply:
I think I could add a generic error, and file a bug for the not found behavior and then ad the not found after the fix, what do you think?

Reply:
Yes, it would be at least what we really return

Reply:
kk

Reply:
done